config/core/test-configs.yaml

```diff
@@ -2546,7 +2565,9 @@ test_configs:
   - device_type: sc7180-trogdor-lazor-limozeen
     test_plans:
       - baseline
+      - baseline-nfs
```

Comment:
This is failing for now as some config options are missing in the arm64-chromebook fragment (see #971):
https://staging.kernelci.org/test/plan/id/62593d43139f8f30b905f4c8/

So we could either drop it from this PR or wait until the missing configs have been added and the whole thing has been tested, as I believe there were other issues too in mainline with this platform.